### DISCUSSION TITLE
Support for Grist

### DISCUSSION THREAD

**Discussion by cdugeai** [evidence-dev/evidence#2417] [Data Source Request]
# Grist

> A modern, open source spreadsheet that goes beyond the grid.

Open source alternative to Airtable/Baserow/Nocodb, see [their website](https://www.getgrist.com/), repo [gristlabs/grist-core | Github](https://github.com/gristlabs/grist-core) (⭐ 6.1K)

## What do you use this database for?

Collaborative database with REST API. Highly customisable and extendable.

## Are there any popular and well-maintained clients for this database?

[REST API Reference](https://support.getgrist.com/api/), or [Typescript SDK](https://www.npmjs.com/package/grist-api)

## Are you interested in contributing this database connector?

Yes, I can give a try !

**Comment by archiewood** [comment]
If you're interested in having a go at this, check out our new docs on creating a plugin connector!

https://docs.evidence.dev/plugins/creating-a-plugin/datasources
